File: 02.PYTHON/pg-166-169/pg_166_11.py
print("")
print("************************************************************")
print("READ TWO INTEGER NUMBERS AND DETERMINE WHICH IS THE GREATEST")
print("************************************************************")
print("")

num1 = int(input("Write the first integer number: "))
num2 = int(input("Wite the second integer number: "))

if(num1 > num2):
    print("The first number is greatest than second.")
elif(num2 >num1):
    print("The second number is greatest than first.")
else:
    print("the number is 0.")

File: 03.JAVASCRIPT/pg-166-169/pg_166_11.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("READ TWO INTEGERS AND DETERMINE WHICH IS GREATER");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("");

let numInput1 = document.querySelector(".num1");
let numInput2 = document.querySelector(".num2");
let btnProof = document.querySelector(".btnProof");
let textar = document.querySelector(".textar");

function validateNumber()
{
    let num1 = Number(numInput1.value);
    let num2 = Number(numInput2.value);

    if(num1 > num2)
    {
        textar.value = "The first number is greater than the second.";
    }
    else if(num2 > num1)
    {
        textar.value = "The second number is greater than the first.";
    }
    else
    {
        textar.value = "Both numbers are the same.";
    }
}

btnProof.addEventListener("click", validateNumber);
numInput1.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});

numInput2.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});